js/script.js

```diff
@@ -35,6 +40,10 @@ const renderTasks = () => {
   tasks.forEach((task) => {
     const texts = [task.title, " Created at: " + task.createdAt];
     const $tasksList = createElement(texts, "li");
+    const $deleteButton = createElement("Delete", "button", () =>
```

Comment:
isn't every delete button supposed to be a part of the task itself? Create the button inside each task and then simply re-render the list.

Reply:
Done, bhai.